b3lb/rest/migrations/0008_add_record_models.py

```diff
@@ -0,0 +1,81 @@
+# Generated by Django 3.2.10 on 2022-01-18 12:57
+
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import rest.models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rest', '0007_alter_parameter_parameter'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='meeting',
+            name='end_callback_url',
+            field=models.URLField(default=''),
+        ),
+        migrations.AddField(
+            model_name='meeting',
+            name='end_nonce',
```

Comment:
I don't know if this is required (see above). Could we add a field description (i.e. _This nonce value protects..._) ?

Reply:
Changed "end_nonce" to "nonce".